tests/test_precision.py

```diff
@@ -78,7 +78,7 @@ def predict_and_measure_memory(precision) -> int:
 def test_train_precision_memory(precision: Precision):
     memory_fp32 = fit_and_measure_memory(Precision.FP32)
     memory_half = fit_and_measure_memory(precision)
-    assert memory_half < 0.87 * memory_fp32
+    assert memory_half < 0.92 * memory_fp32
```

Comment:
just to make sure we're not missing something here...can you try increasing the model size and see if this threshold can be made lower? may or may not merge depending on how long it makes the test take, but would like to know

Reply:
It seems that something is weird here; I can increase the model size to `resnet50`, which gets us below this threshold for training, but for eval and predict, the memory usage is somehow actually higher than with FP32???

Reply:
jk just added some garbage collection and the test seem fine now